src/components/about.tsx

```diff
@@ -0,0 +1,17 @@
+interface AboutProps {
+  name: string;
+  description: string;
+}
+
+const About = ({ name, description }: AboutProps) => {
+  return (
+    <div className="flex flex-col items-center text-center font-Malayalam font-normal">
+      <div className="text-7xl">About</div>
```

Comment:
use vh, vw units instead ex: text-[5vw] here's an example of what they are

https://www.sitepoint.com/css-viewport-units-quick-start/

We'll go over this at tomorrow's meeting